c7n/resources/workspaces.py

```diff
@@ -519,6 +519,45 @@ def process(self, resources, event=None):
         return results
 
 
+@WorkspacesWeb.filter_registry.register('user-access-logging-settings')
```

Comment:
I think we can drop -settings suffix for simplicity.

Reply:
Sounds good